specification/network/resource-manager/Microsoft.Network/stable/2021-05-01/firewallPolicy.json

```diff
@@ -567,6 +567,256 @@
+            "required": true,
+            "description": "Will contain all properties of the object to put",
+            "schema": {
+              "$ref": "#/definitions/SignaturesOverrides"
+            }
+          },
+          {
+            "name": "resourceGroupName",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The name of the resource group."
+          },
+          {
+            "name": "firewallPolicyName",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The name of the Firewall Policy."
+          },
+          {
+            "$ref": "./network.json#/parameters/SubscriptionIdParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Will return the policy current signature overrides object",
+            "schema": {
+              "$ref": "#/definitions/SignaturesOverrides"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "./network.json#/definitions/CloudError"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "put signature overrides": {
+            "$ref": "./examples/FirewallPolicySignatureOverridesPut.json"
+          }
+        }
+      },
+      "get": {
+        "description": "Returns all signatures overrides for a specific policy.",
+        "operationId": "FirewallPolicyIdpsSignaturesOverrides_Get",
+        "parameters": [
+          {
+            "name": "resourceGroupName",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The name of the resource group."
+          },
+          {
+            "name": "firewallPolicyName",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The name of the Firewall Policy."
+          },
+          {
+            "$ref": "./network.json#/parameters/SubscriptionIdParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Will return the policy current signature overrides object",
+            "schema": {
+              "$ref": "#/definitions/SignaturesOverrides"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "./network.json#/definitions/CloudError"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "get signature overrides": {
+            "$ref": "./examples/FirewallPolicySignatureOverridesGet.json"
+          }
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Network/firewallPolicies/{firewallPolicyName}/": {
```

Comment:
>firewallPolicyName}/ [](http://example.com/codeflow?start=118&length=20)

Action name is missing.

Reply:
Added the action name.

Reply:
Why listAllIdpsFilterOptions rather than just listIdpsFilterOptions?

Reply:
done